#!/usr/bin/env python
PACKAGE = "phidgets"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("data_rate",               double_t, 0, "Data rate Hz", 4, 0.017, 20)
gen.add("position_change_trigger", int_t, 0,    "Minimum position change to trigger (0 trigger at data_rate)", 0, 0, 60000000)

exit(gen.generate(PACKAGE, "phidgets", "Encoder"))